src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx

```diff
@@ -61,24 +56,6 @@ export const HeaderSWA = () => {
                   enableNewSubmissionFlow ? "sell2/intro" : "/sell/submission"
                 }
                 onClick={event => {
-                  if (!isLoggedIn) {
```

Comment:
Removing the auth check here will also affect the new Sell flow (`sell2/intro`). This should be fine because the new flow has an additional check on the Artist step.

<img width="863" alt="Screenshot 2024-07-01 at 11 46 31" src="https://github.com/artsy/force/assets/4691889/c10ce068-4ea7-48dc-911a-f5d851dbc3be">